packages/round-manager/src/features/round/__tests__/ApplicationsReceived.test.tsx

```diff
@@ -140,7 +149,63 @@ describe("<ApplicationsReceived />", () => {
       expect(secondApproveButton).toHaveClass("bg-teal-400 text-grey-500")
     });
 
+    describe("when at least one application is selected", () => {
+      it("displays the continue button and copy", () => {
+        renderWrapped(<ApplicationsReceived bulkSelect={true} />)
+
+        const approveButton = screen.queryAllByTestId("approve-button")[0]
+        fireEvent.click(approveButton)
+
+        const continueButton = screen.getByRole('button', {
+          name: /Continue/i
+        });
+        expect(continueButton).toBeInTheDocument();
+        expect(screen.getByText(/You have selected 1 Grant Applications/i)).toBeInTheDocument();
+
+        const approveButton2 = screen.queryAllByTestId("approve-button")[1]
+        fireEvent.click(approveButton2)
+
+        expect(continueButton).toBeInTheDocument();
+        expect(screen.getByText(/You have selected 2 Grant Applications/i)).toBeInTheDocument();
+      })
+
+      it("opens the confirmation modal with the correct number of selected applications when the continue button is clicked", async () => {
```

Comment:
nitpick/my bad: this test technically doesn't verify that the confirmation modal displays the correct number of selected applications (example: "1 approved, 2 rejected")

Reply:
fixed :)